content/ru/tutorials/keepyup-part-three.md

```diff
@@ -138,7 +138,7 @@ this.app.on("game:start", function () {
 
 The game script also manages the current score. It exposes methods that are used to modify the score and also fires events to let other code know that the score has changed.
 
-### Resolution
+### Resolution (Разрешение)
```

Comment:
Why is the translation in brackets?

Reply:
It's because Resolution also appears in the Settings page and since it's something that occurs in the Editor interface, it's been left in English and translated in brackets. But since Transifex plugs in translated phrases across the whole site, it's replaced it here too. In this case, the English doesn't need to be retained and simply the Russian translation is fine. But Transifex isn't smart enough to know that. It's why you have to be a bit careful and check the output from Transifex.

Reply:
it's tedious to go to every file and update it - I would prefer to fix it when I will process them.

Reply:
Fine by me (and yeah, it's extremely tedious!) 😄